content/docs/operating/configuration.md

```diff
@@ -662,6 +662,12 @@ See below for the configuration options for Marathon discovery:
 servers:
   - <string>
 
+# Optional bearer token authentication information.
+  [ bearer_token: <string> ]
```

Comment:
Yes I know, but I used the same indentation, that was needed in the target environment:
```
    marathon_sd_configs:
    - servers:
      - 'https://SERVER/marathon'
      bearer_token_file: token
      tls_config:
        insecure_skip_verify: true
```

Using other indentations neither bearer_token_file nor tls_config are used.

So maybe for clarity I should indent the comments the same way as the options?